client/src/App.tsx

```diff
@@ -0,0 +1,99 @@
+import { FormEvent, useEffect, useState } from "react";
+import "./App.css";
+import { Movie, deleteMovie, getMovies, saveMovie, updateMovie } from "./lib/data";
+import { MovieCard } from "./components/MovieCard";
+import "./movie.css";
+import { MovieForm } from "./components/MovieForm";
+
+function App() {
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  useEffect(() => {
+    async function fetchMovies() {
+      const fetchedMovies = await getMovies();
```

Comment:
Always put try/catch around your async functions!

Also, potential employers will discount your solution if you do not show a "loading" indicator of some sort, or if you do not display an error message to the user. Your code is missing both of those.